cmd/lotus-chainwatch/processor/processor.go

```diff
@@ -94,14 +94,20 @@ func (p *Processor) Start(ctx context.Context) {
 		for {
 			select {
 			case <-ctx.Done():
-				log.Infow("Stopping Processor...")
+				log.Debugw("Stopping Processor...")
 				return
 			default:
 				toProcess, err := p.unprocessedBlocks(ctx, p.batch)
 				if err != nil {
 					log.Fatalw("Failed to get unprocessed blocks", "error", err)
 				}
 
+				if len(toProcess) == 0 {
+					log.Debugw("No unprocessed blocks. Wait then try again...")
+					time.Sleep(time.Second * 10)
```

Comment:
commentary: we could also use [`EpochDurationSeconds`](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/network.go#L12) here